crates/ruff_python_ast/src/nodes.rs

```diff
@@ -3147,6 +3157,20 @@ pub struct ParameterWithDefault {
     pub default: Option<Box<Expr>>,
 }
 
+impl ParameterWithDefault {
+    pub fn default(&self) -> Option<&Expr> {
+        self.default.as_deref()
+    }
+
+    pub const fn name(&self) -> &Identifier {
+        self.parameter.name()
+    }
+
+    pub fn annotation(&self) -> Option<&Expr> {
+        self.parameter.annotation()
+    }
```

Comment:
I considered implementing `Deref<Target = Parameter>` for `ParameterWithDefault` so that these methods would be implemented "for free", but decided against it. A `ParameterWithDefault` does more than simply wrap a `Parameter`: for example, if its `default` field is not `None`, it will have a different `range` value to the `range` of its inner `parameter` field.